wolox-training/rof-angular
# Implemented book detail view

## Summary

Implemented book detail view

## Test Plan

* Log into your account
* Access the books' dashboard
* Click on any of their titles
* Confirm you can access de book detail

## Trello Card 

 https://trello.com/c/rSw7UbQJ

## Screenshot

![detail](https://cloud.githubusercontent.com/assets/2911971/10743979/4f201404-7c15-11e5-8fe7-1233ef75bdf1.png)
